engine/collection/ingest/engine.go

```diff
@@ -222,7 +222,7 @@ func (e *Engine) onTransaction(originID flow.Identifier, tx *flow.TransactionBod
 	// if the message was submitted internally (ie. via the Access API)
 	// propagate it to members of the responsible cluster (either our cluster
 	// or a different cluster)
-	if originID == e.me.NodeID() {
+	if originID == e.me.NodeID() && len(txCluster.NodeIDs()) > 1 {
```

Comment:
on a localnet with just 1 collection node, the `conduit.Multicast` throws an error - `failed to multicast on channel push-transactions: empty list of target ids` since the collection node tries to send the tx to _other_ collection nodes that don't exist.
This change skips that send if there is only 1 collection node

Reply:
This won't work in the case where we have multiple clusters with one node each, and the node in cluster 1 receives a transaction for cluster 2.

```suggestion
	if originID == e.me.NodeID() {
		recipientIDs := txCluster.NodeIDs().Filter(filter.Not(filter.HasNodeID(e.me.NodeID())))
		if len(recipientIDs) == 0 {
			return nil
		}
```